README.md

```diff
@@ -149,10 +149,67 @@ Returns the `webhooks` API.
 ### webhooks.sign()
 
 ```js
-webhooks.sign(eventPayload);
+webhooks.sign(secret, eventPayload);
 ```
 
 <table width="100%">
+  <tr>
+    <td>
+      <code>
+        secret
+      </code>
+      <em>
+        (string)
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      secret to generated the signature
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>
+        eventPayload
+      </code>
+      <em>
+        (Object)
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong>
+      Webhook request payload as received from GitHub
+    </td>
+  </tr>
+</table>
+
+or
+
+```js
+webhooks.sign({
+  secret: <my_awesome_secret>,
+  algorithm: <"sha1" | "sha256">
+}, eventPayload);
+```
+
+<table width="100%">
+  <tr>
+    <td>
+      <code>
+        options
+      </code>
+      <em>
+        (Object)
+      </em>
+    </td>
+    <td>
+      <strong>Required.</strong> secret to generated the signature
+      algorithm: secret to generated the signature
+    </td>
+    <td>
+      algorithm to be applied when encrypting (sha1 by default)
+    </td>
```

Comment:
@gr2m What do you think?

Reply:
I'd list the two call signatures on top and then have a single table, like this:

<details>

````md
```js
webhooks.sign(secret, eventPayload);
webhooks.sign({ secret, algorithm }, eventPayload);
```

<table width="100%">
  <tr>
    <td>
      <code>
        secret
      </code>
      <em>
        (string)
      </em>
    </td>
    <td>
      <strong>Required.</strong>
      secret to generated the signature
    </td>
  </tr>
  <tr>
    <td>
      <code>
        algorithm
      </code>
      <em>
        (string)
      </em>
    </td>
    <td>
      Algorithm to calculate signature. Can be set to `sha1` or `sha256`. Defaults to `sha1`.
    </td>
  </tr>
  <tr>
    <td>
      <code>
        eventPayload
      </code>
      <em>
        (Object)
      </em>
    </td>
    <td>
      <strong>Required.</strong>
      Webhook request payload as received from GitHub
    </td>
  </tr>
</table>
````

</details>